ext/openssl/openssl.c

```diff
@@ -127,6 +128,116 @@ enum php_openssl_encoding {
 	ENCODING_PEM,
 };
 
+/* X509Certificate class */
+
+static zend_object_handlers x509_certificate_object_handlers;
+
+static zend_object *x509_certificate_create_object(zend_class_entry *class_type) {
```

Comment:
please prefix all functions with `php_openssl_` which is convention in this file to prevent any potential conflicts with OpenSSL functions - I know that this one won't conflict but it's good to have it the same. OpenSSL also has some functions that are exported and are pretty generic so I chose to use this.

Reply:
you can shorten the rest of the name of course. For example in this case it could be just something like `php_openssl_x509_create_object`.